### DISCUSSION TITLE
Programmatically compile: how to substitute Deno.emit?

### DISCUSSION THREAD

**Discussion by retog** [denoland/deno#14755] [Q&A] [answered]
With deno 1.22 [deno.emit](https://doc.deno.land/deno/unstable@v1.21.0/~/Deno.emit) is no longer available. What's the current way to programmatically execute the functionality provided by the command-line tool, such as compiling and bundling?

**Comment by bartlomieju** [comment, marked as answer]
See https://github.com/denoland/deno_emit

**Reply by retog** [reply]
Odd having to download a library for a functionality that's already in deno, but at least a way to get my code working again. Thanks!
